server/ldap_check.rb

```diff
@@ -86,6 +86,8 @@ def is_user_in_admin_hash user
       @updateCount += 1
       @lastUpdate = Time.now()
       save_group_members configuration['group']
+      # Might get a group with no members or that doesn't exist.
+      @admin_hash = Hash.new unless(@admin_hash)
     end
 
     @admin_hash.has_key? user
```

Comment:
Admittedly, I don't know Ruby syntax very well, but rather than declaring an empty hash above, couldn't this test be written as (untested pseudo-code):

    @admin_hash && (@admin_hash.has_key? user)